Makefile

```diff
@@ -12,14 +12,13 @@ SRCS = $(wildcard API/*.c contrib/**/*.c)
 OBJS = $(patsubst %.c,%.o,$(SRCS))
 CFLAGS += -fno-strict-aliasing -fwrapv
 CFLAGS += -Wall -Wextra -Wpointer-sign -Wno-unused-parameter
-CFLAGS += -D_GNU_SOURCE=1
 
 .DEFAULT: libev3api.a
 libev3api.a: $(OBJS)
 	$(AR) rcs $@ $^
 
 %.o: %.c
-	$(CC) -Os $(CFLAGS) -isystem. -c $< -o $@ 
+	$(CC) -std=c99 -D_GNU_SOURCE=1 -Os $(CFLAGS) -isystem. -c $< -o $@

```

Comment:
Why move _GNU_SOURCE out of `CFLAGS`?

Reply:
This is because `.travis.yml` overwrites CFLAGS and therefore the macro is not included anymore.

Reply:
Then the Makefile should be changed to append to CFLAGS instead, no?

Reply:
Fixed in b727cf0f2c1ba9e6e15dde34e12745ace6bf54a0 + bd3f511ad885f2d3a6afa29b30bc542d737d2535 :+1:

Reply:
I can split this into a separate PR so that only one area is addressed in this PR.

Reply:
Split into #46

Reply:
The build of this PR is expected to fail on the old compiler now, because it uses C89 by default.